gnuradio-runtime/lib/pmt/pmt_int.h

```diff
@@ -128,6 +128,15 @@ class pmt_pair : public pmt_base
     void set_cdr(pmt_t cdr) { d_cdr = cdr; }
 };
 
+class pmt_dict : public pmt_pair
+{
+public:
+    pmt_dict(const pmt_t& car, const pmt_t& cdr);
+    //~pmt_dict(){};
```

Comment:
If we don't need, it remove it please.

Reply:
the rest of the classes in this file have commented destructors, should they all be removed? I wasn't sure why so i put this one in.

Reply:
consistency is nice too so...meh don't care.